### PR TITLE
Kill Enemies

### PR DESCRIPTION
**Pull request by tikokito123**
That was a headache...

### CODE REVIEW

File: bpdv3/Assets/Scripts/Floor/OnTile.cs [hunk 1]
```diff
@@ -28,15 +28,15 @@ public class OnTile : MonoBehaviour
 
         private void Update()
         {
```
Comment (RSGameDev) [inline bpdv3/Assets/Scripts/Floor/OnTile.cs:30]: This looks like the enemy movement will not work as it did with this code commented out.
Can you give some information as to why this was commented out please.

**Review by RSGameDev — COMMENTED** [review]
Thanks. There was 1 comment i left if you would be able to let me know your thoughts on.

File: bpdv3/Assets/Scripts/Floor/OnTile.cs (hunk 1 above)
Comment (tikokito123) [reply bpdv3/Assets/Scripts/Floor/OnTile.cs:30]: The enemy movement did work on my pc withou this

Comment (RSGameDev) [reply bpdv3/Assets/Scripts/Floor/OnTile.cs:30]: thanks for the feedback

**Review by RSGameDev — APPROVED** [review]
Thanks for working on this feature. This can be merged with dev branch now.

**Pull request merged**